django_celery_results/settings.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+
+
+def get_callback_function(settings_name, default=None):
+    """Return the callback function for the given settings name."""
+    
+    callback = getattr(settings, settings_name, None)
+    if callback is None:
+        return default
+
+    if callable(callback):
+        return callback
+
+extend_task_props_callback = get_callback_function(
```

Comment:
Consider adding some sanity checks on the return value of the callback.
For example that it complies to the `Mapping` protocol.

Reply:
Well, `get_callback_function` is quite generic, and could be used in the future for another purposes, returning different types of data. So far I can see, the sanity checks could be in `_store_results()` where `extend_task_props_callback()` is called. What do you think?

Reply:
I'm personally not a fan of this generic function, I'd err of the side of caution and make the callback handling as clean and explicit as possible so any errors we need to raise have a clear source.
But this is more of a code-style thing so maybe one of the maintainers (@auvipy) can comment.

Reply:
Good point @AllexVeldman, what do you think @auvipy?

Reply:
@AllexVeldman & @auvipy I think I found a proper way to control the callback internally being able to check explicitly that return value.

I just created a new function called `get_task_props_extension()` into the settings module in charge to return an empty dict in case that the `CELERY_RESULTS_EXTEND_TASK_PROPS_CALLBACK` is undefined and otherwise will check that the return value complies with the Mapping protocol.

Let me know what you think!

Reply:
LGTM